### PR TITLE
Story 6

### PR DESCRIPTION
**Pull request by zdrvk**

### CODE REVIEW

File: src/Components/Bookpage/BookDetails/index.js [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+import Reviews from "./Reviews"
+import "./bookdetails.css"
+
+function BookDetails({ image, title, author, year, pageCount, genre, blurb, reviews }) {
+    const totalScore = reviews.reduce((ratingSum, review) => ratingSum + review.rating, 0)
+    const avgScore = totalScore / reviews.length
+
 import ClaimForm from './ClaimForm'
 import ReturnForm from './ReturnForm'
 import './bookdetails.css'
```
Comment (ashleycoles) [inline src/Components/Bookpage/BookDetails/index.js:12]: This section is breaking the whole app. imports need to be at the top - looks like a merge conflict that's been messed up slightly

File: src/Components/Bookpage/BookDetails/Reviews/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import "./reviews.css"
+
+function Reviews({ review }) {
```
Comment (ashleycoles) [inline src/Components/Bookpage/BookDetails/Reviews/index.js:3]: Rename to 'Review' since this is singular

**Review by ashleycoles — CHANGES_REQUESTED** [review]
It's broken! I can't get the app to run, getting an error from the bookDetails component. Otherwise it's good

File: src/Components/Bookpage/BookDetails/index.js (hunk 1 above)
Comment (vladi-stanchev) [reply src/Components/Bookpage/BookDetails/index.js]: fixed

File: src/Components/Bookpage/BookDetails/Reviews/index.js (hunk 2 above)
Comment (vladi-stanchev) [reply src/Components/Bookpage/BookDetails/Reviews/index.js]: done

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**